test/test_pynisher.py

```diff
@@ -74,7 +74,8 @@ def spawn_rogue_subprocess(num_procs=5):
 def simulate_work(size_in_mb, wall_time_in_s, num_processes, **kwargs):
     # allocate memory (size_in_mb) with an array
     # note the actual size in memory of this process is a little bit larger
-    A = [42.] * ((1024 * size_in_mb) // 8) # noqa
+    FLOAT_MAX = 3.402823466E+38
```

Comment:
Hey, do you mean that this is the maximum number of float64 or float 128? Because otherwise you could use `sys.float_info.max` which is substantially higher than your number.

Reply:
Yes, the maximum 64 bit float number.

Reply:
Could you please add that as an explanation?